src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -44,6 +44,9 @@
     ROUTE_UNUSED(Priority.HIGH, "This route is defined but has no trips."),
     SERVICE_NEVER_ACTIVE(Priority.MEDIUM, "A service code was defined, but is never active on any date."),
     SERVICE_UNUSED(Priority.MEDIUM, "A service code was defined, but is never referenced by any trips."),
+    SERVICE_WITHOUT_DAYS(Priority.MEDIUM, "At least one of the days of the week fields (Monday,\n" +
+        "Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday) must have its value set to '1'\n" +
+        "for each service defined in 'calendars.txt'."),
```

Comment:
I don't understand why there are new lines in this description string. Also, a slight revision to bring it in line with other error descriptions:

> A service defined in calendar.txt should be active on at least one day of the week. Otherwise, it should be omitted from this file.

Reply:
I pasted the text from the MTC guidelines, then IntelliJ split the lines automatically.